src/components/EnvironmentOverride/DeploymentTemplateOverrideForm.tsx

```diff
@@ -703,6 +735,11 @@ export default function DeploymentTemplateOverrideForm({
                 componentType={3}
                 setShowLockedDiffForApproval={setShowLockedDiffForApproval}
                 setLockedConfigKeysWithLockType={setLockedConfigKeysWithLockType}
+                lockedConfigKeysWithLockType={lockedConfigKeysWithLockType}
+                setHideLockedKeys={setHideLockedKeys}
+                hideLockedKeys={hideLockedKeys}
+                hideLockKeysToggled={hideLockKeysToggled}
+                inValidYaml={state.unableToParseYaml}
```

Comment:
We should use it from the context instead of passing

Reply:
ignored as its in fe-lib